pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -332,8 +334,29 @@ func (k *Bootstrapper) WaitForNode(cfg config.ClusterConfig, n config.Node, time
 		glog.Infof("%s is not a control plane, nothing to wait for", n.Name)
 		return nil
 	}
+
+	// starting from minikube v1.10 checking for node pressure is a reuqired check
+	out.T(out.HealthCheck, "Verifying Kubernetes Components:")
+	out.T(out.OptionVerify, "verifying node conditions ...")
+
+	// TODO: #7706: for better perforamnce we could use k.client inside minikube to avoid asking for external IP:PORT
```

Comment:
This comment doesn't make sense yet, since you call `k.client` just below it. Also, check comment spelling.